src/string/memset.c

```diff
@@ -0,0 +1,13 @@
+#include <string.h>
+
+void
+memset(void *s, int c, size_t n)
+{
+	char *cs = (char *)s;
+	unsigned char cc = (unsigned char)c;
+	size_t i = 0;
+	for (; i < n; i++)
+	{
+		cs[i] = cc;
+	}
```

Comment:
a blank line is needed between declaration and the rest of the program (here between line 8-9) also function must return a `void *`

also `i` is not needed here, the function can be simplified:
```c
void *
memset(void *s, int c, size_t n)
{
    unsigned char *cs ;
    
    cs =(unsigned char *)s;
    while (i--)
    {
        cs++ = (unsigned char)c;
    }
    return (s);
}
```